FlingEngine/Graphics/inc/FlingWindow.h

```diff
@@ -4,6 +4,8 @@
 #include "FlingTypes.h"
 #include "NonCopyable.hpp"
 
+enum WindowMode { Fullscreen, Windowed, BorderlessWindowed };
```

Comment:
I would prefer this to be on new lines (i.e: 
```C
enum WindowMode
{
   FullScreen,
   Windowed,
   BorderlessWindowed
};
```

Reason being that it's more readable, and brackets on newlines is like the only standard that I actually enforce(my bad, I should have mentioned it)